Describe Performance Tracing in the Performance Page (#270)
microsoft/TypeScript-wiki: Performance.md

This is rare, but can happen.
Upgrading to a newer version of TypeScript (which can be more efficient) or to a newer version of an `@types` package (which may have reverted a regression) can often solve the issue.

## Performance Tracing

> You might find that these diagnostic methods are too opaque to get a sense of why TypeScript feels slow.

Reviewer: I don't know what "opaque" means in this context, but it feels like "vague" or "imprecise" might be more appropriate?

Author: ```suggestion
In some cases, the approaches above might not give you enough insight to understand why TypeScript feels slow.
```

> TypeScript provides a `--generateTrace` option that can give you a sense of the work the compiler is spending time on.

Author: ```suggestion
TypeScript 4.1 and higher provides a `--generateTrace` option that can give you a sense of the work the compiler is spending time on.
```

`--generateTrace` will create an output file that can be analyzed within Edge or Chrome.

> Make sure TypeScript can compile your project without errors.

Reviewer: It actually doesn't matter if there are errors (though you'll be measuring something different).

Author: ```suggestion
Ideally, TypeScript will be able to compile your project without any errors, though it's not a strict requirement for tracing.
```

> Once you can get a clean compile/type-check, you can run with the `--generateTrace` flag to specify an output directory for tracing.

Author: ```suggestion
Once you're ready to get a trace, you can run TypeScript with the `--generateTrace` flag.
```

* Ideally, a project that demonstrates the issues being encountered.
* Output logs from profiling the compiler (`isolate-*-*-*.log` and `*.cpuprofile` files)

### Providing Performance Traces

[Performance traces (described above)](#performance-tracing) are meant to help teams figure out build performance issues in their own codebases;
however, they can also be useful for the TypeScript team in diagnosing and fixing issues.

> [See the above section on performance traces](#performance-tracing) and continue reading more on [our dedicated performance tracing page](https://github.com/microsoft/TypeScript/wiki/Performance-Tracing).

Reviewer: Those are some big link spans. 😛
